modules/bcftools/reheader/meta.yml

```diff
@@ -0,0 +1,51 @@
+name: bcftools_reheader
+description: Reheader a VCF file
+keywords:
+  - reheader
+  - vcf
+  - update header
+tools:
+  - reheader:
+      description: |
+        Modify header of VCF/BCF files, change sample names.
+      homepage: http://samtools.github.io/bcftools/bcftools.html
+      documentation: http://samtools.github.io/bcftools/bcftools.html#reheader
+      doi: 10.1093/gigascience/giab008
+      licence: ['GPL']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: VCF/BCF file
+      pattern: "*.{vcf.gz,vcf,bcf}"
+  - fai:
+      type: file
+      description: Fasta index to update header sequences with
+      pattern: "*.{fai}"
+  - header:
+      type: file
+      description: New header to add to the VCF
+      pattern: "*.{hr}"
```

Comment:
I was not sure on this extension, technically this can be anything and maybe even `.txt` or `.vcf`?

Reply:
I would probably call it `*header.txt`

Reply:
Wouldn't just `.txt` be safer, if assuming the pipeline user will customise the output file with `$options.suffix`?

Reply:
The problem here is that a `.txt` file could be anything and in theory could be overwritten by a similar named `.txt` file from another module which is why adding the `.header.txt` clarifies it's a file containing the header. This doesn't stop developers from adding their own custom `prefix` though e.g. `sample1.header.txt` / `sample1.filtered.header.txt` etc.